src/sparseml/integration_helper_functions.py

```diff
@@ -47,24 +51,46 @@ def resolve_integration(
     :param source_path: The path to the PyTorch model to export.
     :param integration: Optional name of the integration to use. If not provided,
         will attempt to infer it from the source_path.
+    :param task: Optional name of the task to use.
     :return: The name of the integration to use for exporting the model.
     """
 
     if integration is not None:
         integration = integration.replace("_", "-")
 
+    if task is not None:
+        task = task.replace("_", "-")
```

Comment:
shouldn't this be resolved at a more consolidated level?

Reply:
Like in the `export.py`?